src/components/Person/Person.jsx

```diff
@@ -1 +1,37 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = () => (
+    sex === 'm'
+      ? 'wife'
+      : 'husband'
+  );
```

Comment:
is there a purpose for making it a function?
```suggestion
  const partner = sex === 'm'
    ? 'wife'
    : 'husband';
```

Reply:
I suppose I could make it as 2 variables, one for sex === m and one for sex === f, but the function is shorter I believe